Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -757,16 +757,34 @@ function testNetworkConnectivity {
     # number of URL's that can fail the connectivity before the agent refuses to install (the test fails incorrectly sometimes, so 1 failure is acceptable)
     $connectivityTolerance = 1
 
-    $URLs = @("huntress.io", "huntresscdn.com", "update.huntress.io", "eetee.huntress.io", "huntress-installers.s3.amazonaws.com", "huntress-updates.s3.amazonaws.com", "huntress-uploads.s3.us-west-2.amazonaws.com",
-              "huntress-user-uploads.s3.amazonaws.com", "huntress-rio.s3.amazonaws.com", "huntress-survey-results.s3.amazonaws.com")
+    $URLs = @("https://bugsnag.com",
+	"https://eetee.huntress.io/96bca0cef10f45a8f7cf68c4485f23a4",
```

Comment:
I don't know that we need to bother with testing bugsnag connectivity in the installer script; it's not required for normal operation.

```suggestion
    $URLs = @("https://eetee.huntress.io/96bca0cef10f45a8f7cf68c4485f23a4",
```

Reply:
We added it more recently because people were blocking bugsnag at the firewall and we weren't able to see obvious issues. This is more of a dumb smoke test that the domain isn't blocked.

Reply:
I'll remove it from this tho since it's not quite required